### DISCUSSION TITLE
How to use Supabase .backups files?

### DISCUSSION THREAD

**Discussion by camilopenalver** [supabase/supabase#29964] [Questions]
As many of you know, when we have a pro plan on Supabase, it generates .backup files that we can download. These .backup files contain SQL code, but my question is: how do we actually use them? I can't find any information on how to restore my database from a .backup file. Yes, there is a "download" button in the graphical interface, but how exactly can we use these downloaded backup files to restore the entire database in a new Supabase project, for example, or in another environment?

![image](https://github.com/user-attachments/assets/c51978de-4efb-4139-9582-499b25c47fd7)

**Comment by silentworks** [comment]
There is a guide on how to migrate a project on Supabase https://supabase.com/docs/guides/platform/migrating-within-supabase. The .backup files can be used if you want to migrate away from Supabase and can also be used when you access the database directly to restore it (more advanced feature).
